main.html

```diff
@@ -5,50 +5,53 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href="main.css" rel="stylesheet">
-    <title>Presentation</title>
-    <style>
-        @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap');
-        </style>
+    <title>Document</title>
 </head>
 <body>
-    <div class="top-navigation">
-        <div class="item-navigation">
-            <div class="frame-4">
-                <p>Portfolio</p>
-            </div>
-            <div class="frame-3">
-                <p>About</p>
-            </div>
-            <div class="frame-5">
-                <p>Contact</p>
-            </div>
+    <div class="container">
+    <header class="my-header">
+        <div class="time">
+            <img src="mobile_menu_images/Bars/Status-Bars/Time-Style.jpg" alt="Header image">
         </div>
-        <div class="logo-placeholder">
-            <p>My Logo</p>
+        <div class="status-bar">
+            <img src="mobile_menu_images/Bars/Status-Bars/Group3.jpg" alt="Bar Status icons">
         </div>
-    </div>
+    </header>
     
-        <div class="social-media-icons">
-            <img src="images/social-media-icons.jpg">
+    <div class="rectangle"><img src="mobile_menu_images/Bars/Status-Bars/Rectangle.jpg" alt="Divider"></div>
+
+    <nav class="main-menu">
+      <div class="menu-container">
+        <div class="logo-frame">
+            <p>SIKO NORBERT</p>
         </div>
-        <div class="contact-form-background-shapes">
-            <img src="images/contact-form-background-shapes@2x.jpg">
+        <img src="mobile_menu_images/Normal-Button/Tertiary/Enabled.jpg" alt="Button">
+      </div>
+    </nav>
+
+    <section class="headline">
+        <div class="first-text">
+            <!---  I used without tag because the tag put automatically the margin  --->
+            <h1>I’m Norbert. <br>Glad to see you!</h1>
         </div>
-        <div class="header-bg">
-            <img src="images/Header-bg@2x.jpg">
+        <div class="second-text">
+            <!---  I used without tag because the tag put automatically the margin  --->
+        <h1>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</h1>
```

Comment:
- [ ] The <p> tag is used to structure text in paragraphs within an HTML document. At first, all the texts should be within this tag, except When the text is a title, in this new condition the tags `<h1>` a `<h6>` must be used. In this case, It would be good if you replace your `<h1>` tag with the `p` tag for the reasons addressed above.